src/Momento.Sdk/Auth/StringMomentoTokenProvider.cs

```diff
@@ -17,6 +17,8 @@ public class StringMomentoTokenProvider : ICredentialProvider
     public string ControlEndpoint { get; private set; }
     /// <inheritdoc />
     public string CacheEndpoint { get; private set; }
+    /// <inheritdoc />
+    public string TokenEndpoint { get; private set; }
```

Comment:
where does this get set?

Reply:
Good catch, thanks. It gets set in b75af12 :-)